tests/func/test_data_cloud.py

```diff
@@ -21,7 +21,17 @@
 
 all_clouds = [
     pytest.lazy_fixture(cloud)
-    for cloud in ["s3", "gs", "azure", "gdrive", "ssh", "http", "hdfs"]
+    for cloud in [
+        "s3",
+        "real_s3",
+        "real_oss",
```

Comment:
Let's not add these ones, just to avoid confusing people. The only reason why we still have `real_s3` is that there is 1 test in which we run into moto's bug, so kinda have to use it there for now :slightly_smiling_face: 
```suggestion
```